android/src/com/mapswithme/maps/routing/SlotFrame.java

```diff
@@ -99,7 +100,7 @@ private void updateText()
       if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
         mText.setText(R.string.p2p_your_location);
       else
-        mText.setText(mMapObject.getTitle());
+        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
```

Comment:
Dropped Pin будет приходить из ядра в Title. Subtitle будет пустой.

> On 25.3.2016, at 21.41, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/routing/SlotFrame.java:
> 
> > @@ -99,7 +100,7 @@ private void updateText()
> >        if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
> >          mText.setText(R.string.p2p_your_location);
> >        else
> > -        mText.setText(mMapObject.getTitle());
> > +        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
> А вот это плохо. Раньше же всегда было "Dropped pin".
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>